internal/process/process.go

```diff
@@ -187,5 +186,5 @@ func isQuitting(ctx context.Context) bool {
 }
 
 func defaultReadPipe(r io.Reader) {
-	io.Copy(ioutil.Discard, r)
+	io.Copy(io.Discard, r)
```

Comment:
@powersj Hi, thanks for taking the time to review this PR. I believe these lint errors have existed before this PR, and they are reported because lines like these have been modified in this PR.

To make the lint passes, the easiest way is to ignore the error. I'll force push a new commit now.

Reply:
Just a side note, the screenshot below shows some of the lint errors reported by `golangci-lint`. These errors will be ted by the `lint-pr-changes` job when the corresponding lines are modified.
![2021-09-24_02-38](https://user-images.githubusercontent.com/20135478/134564827-d54fcb90-d108-4210-aca6-80a0c7a4184c.png)